<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myteam</class>
 <widget class="QDialog" name="myteam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1294</width>
    <height>859</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">myteam{
	background-image: url(:/background/fantasy fielde dited.png);
}</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1311</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <item>
     <widget class="QPushButton" name="LeaguesButton">
      <property name="styleSheet">
       <string notr="true">
color: rgb(55, 0, 60);
background-color: rgb(0, 255, 135);
border-radius-top: 20px;</string>
      </property>
      <property name="text">
       <string>Leagues</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="styleSheet">
       <string notr="true">
color: rgb(55, 0, 60);
background-color: rgb(0, 255, 135);
border-radius-top: 20px;</string>
      </property>
      <property name="text">
       <string>My Team</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="styleSheet">
       <string notr="true">
color: rgb(55, 0, 60);
background-color: rgb(0, 255, 135);
border-radius-top: 20px;</string>
      </property>
      <property name="text">
       <string>Logout</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>60</y>
     <width>171</width>
     <height>57</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="goalkeeperLabel">
      <property name="text">
       <string>goalkeeper</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="goalkeeperComboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>210</y>
     <width>171</width>
     <height>57</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="secondndDefenderLabel">
      <property name="text">
       <string>defender</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="secondDefenderComboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>210</y>
     <width>171</width>
     <height>57</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="thirdDefenderLabel">
      <property name="text">
       <string>defender</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="thirdDefenderComboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>340</y>
     <width>171</width>
     <height>57</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QLabel" name="secondMidfielderLabel">
      <property name="text">
       <string>midfielder</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="secondMidfielderComboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>340</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="firstMidfielderLabel">
      <property name="text">
       <string>midfielder</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="firststMidfielderComboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_7">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>500</y>
     <width>201</width>
     <height>57</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QLabel" name="attackerLabel">
      <property name="text">
       <string>attacker</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="attackerComboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>70</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>save team</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>210</y>
     <width>201</width>
     <height>57</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="firstDefenderLabel">
      <property name="text">
       <string>defender</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="firstDefenderComboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>331</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="fantasyTeamNameLabel">
      <property name="text">
       <string>Fantasy Team Name</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="fantasyTeamNameLineEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
